<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="home-laptop" failures="1" tests="6" name="Test" time="34.567" errors="0" timestamp="2023-09-26T20:55:51 IST">
  <testcase classname="com.contactslist.api.tests.UserApiTests" name="addNewUser" time="24.322"/>
  <testcase classname="com.contactslist.api.tests.UserApiTests" name="getUserProfile" time="1.817"/>
  <testcase classname="com.contactslist.api.tests.UserApiTests" name="updateUser" time="1.435"/>
  <testcase classname="com.contactslist.api.tests.UserApiTests" name="userLogout" time="1.279"/>
  <testcase classname="com.contactslist.api.tests.UserApiTests" name="userLogin" time="1.217"/>
  <testcase classname="com.contactslist.api.tests.UserApiTests" name="deleteUser" time="1.293">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.contactslist.api.tests.UserApiTests.deleteUser(UserApiTests.java:101)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- deleteUser -->
</testsuite> <!-- Test -->
